### PR TITLE
Update additional JPA FATs to use test framework library project.

### PR DESCRIPTION
**Pull request by jgrassel**
Update additional JPA FAT buckets to use the test framework library project as well as more clearly passing Exceptions to junit runner.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFINoInheritance/META-INF/permissions.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFINoInheritance/META-INF/permissions.xml:4]: Copyright should include 2 years.... 1st year delivered, and last year modified (unless same as 1st year).... so this should be :
```
Copyright (c) 2019, 2020 IBM Corporation and others.
```

Probably true for most of the files where you've changed the copyright.

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFIYesInheritance/META-INF/permissions.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFIYesInheritance/META-INF/permissions.xml:4]: same as above

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFIYesInheritanceDDOvrd/META-INF/permissions.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFIYesInheritanceDDOvrd/META-INF/permissions.xml:4]: same as above

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDMINoInheritance/META-INF/permissions.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDMINoInheritance/META-INF/permissions.xml:4]: same as above

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDMIYesInheritance/META-INF/permissions.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDMIYesInheritance/META-INF/permissions.xml:4]: same as above

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDMIYesInheritanceDDOvrd/META-INF/permissions.xml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDMIYesInheritanceDDOvrd/META-INF/permissions.xml:4]: same as above

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionJNDI/META-INF/permissions.xml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionJNDI/META-INF/permissions.xml:4]: same as above

File: dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/web/injectionDFINoInheritance/META-INF/permissions.xml [hunk 8]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/web/injectionDFINoInheritance/META-INF/permissions.xml:4]: same as above

**Review by tkburroughs — APPROVED** [review]

**Pull request merged**